tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py

```diff
@@ -206,12 +206,11 @@ def get_change_info(change_id, config):
 
 def get_lp_bug_info(bug_id):
     bug_url = 'https://api.launchpad.net/1.0/bugs/{}'.format(bug_id)
-    bug = {"description": "Not found", "title": "Not found"}
+    bug = {"description": "Not found"}
```

Comment:
Maybe delete LP function altogether - there are no plans to go back to LaunchPad bugs.